specs/refactor_gluster_integration_get_state_dump.rst

```diff
@@ -0,0 +1,119 @@
+================================================
+Refactor Gluster Integration's state dump module
+================================================
+
+The gluster get-state output is consumed by tendrl-gluster-integration, we
+need to refactor the way Tendrl consumes and processes the get-state output
+
+Notes:
+
+* In light of new changes to gluster get-state
+  http://review.gluster.org/#/c/15889
+
+Problem description
+===================
+
+Current Tendrl way of consuming and processing is not robust enough to adopt
+with the Tendrl object namespacing, We also need to accomodate lates
+changes to the gluster get-state output.
+
+Use Cases
+=========
+
+This addresses the use case of obtaining refereshed gluster cluster state
+
+Proposed change
+===============
+
+* General re-factoring of code handling gluster get-state
+
+* Accomodate latest changes to output of gluster get-state w.r.t volume options
```

Comment:
Probably more details are required here.  Current format(provided by glusterd) and the expected change